src/dashboard.Dockerfile

```diff
@@ -34,8 +34,6 @@ ADD dashboard/src/requirements/ /src/dashboard/src/requirements/
 RUN pip install -r /src/dashboard/src/requirements/production.txt -r /src/dashboard/src/requirements/dev.txt
 ADD dashboard/ /src/dashboard/
 
-ADD archivematicaCommon/etc/dbsettings /etc/archivematica/archivematicaCommon/dbsettings
-
 RUN set -ex \
 	&& groupadd -r archivematica \
 	&& useradd -r -g archivematica archivematica
```

Comment:
Of course, that is probably the better way to do it since collectstatic doesn't need a valid value :+1: